### PR TITLE
fix(WEBRTC-149): Enabled multiple connections in multiple devices (browsers, tabs, and etc)

### PR DESCRIPTION
**Pull request by DeividVeloso**
- Changed local storage to use session storage on the Login class to save the `sessid`

## 📝 To Do

- [x] All linters pass
- [x] All tests pass
- [ ] Change documentation based on my changes

## ✋ Manual testing

1. cd `packages/js` && `yarn install` && `yarn build`
2. navigate into `examples/vanilla`
3. open `index.html` in two tabs
4. register the same user in both tabs and see if they keep connected in both tabs.
5. Make a call and check if both tabs received the call.

### Desktop

- [ ] Edge (latest)
- [x] Chrome
- [x] Firefox
- [x] Safari

### CODE REVIEW

File: packages/js/src/Modules/Verto/index.ts [hunk 1]
```diff
@@ -46,14 +46,24 @@ export default class Verto extends BrowserSession {
...
+      login_token,
+      this.sessionid,
+      userVariables
+    );
     const response = await this.execute(msg).catch(this._handleLoginError);
     if (response) {
       this._autoReconnect = true;
       this.sessionid = response.sessid;
-      localStorage.setItem(SESSION_ID, this.sessionid);
+      sessionStorage.setItem(SESSION_ID, this.sessionid);
```
Comment (DeividVeloso) [inline packages/js/src/Modules/Verto/index.ts:66]: The change was made here.  
The rest is ESLint.

File: packages/js/src/Modules/Verto/BrowserSession.ts [hunk 2]
```diff
@@ -63,14 +77,17 @@ export default abstract class BrowserSession extends BaseSession {
   }
 
   async connect(): Promise<void> {
-    this.sessionid = await localStorage.getItem(SESSION_ID);
+    this.sessionid = await sessionStorage.getItem(SESSION_ID);
```
Comment (DeividVeloso) [inline packages/js/src/Modules/Verto/BrowserSession.ts:80]: The change was made here.  
The rest is ESLint.

File: packages/js/examples/vanilla/index.html [hunk 3]
```diff
@@ -224,6 +224,7 @@ <h5>Remote Video</h5>
           login: document.getElementById('username').value,
           password: document.getElementById('password').value,
           ringtoneFile: './sounds/incoming_call.mp3',
+          // iceServers: [{ urls: ['stun:stun.l.google.com:19302'] }],
```
Comment (DeividVeloso) [inline packages/js/examples/vanilla/index.html:227]: Added to exemplify

File: packages/js/src/Modules/Verto/BrowserSession.ts (hunk 2 above)
Comment (hugobdev) [reply packages/js/src/Modules/Verto/BrowserSession.ts:80]: Could we run ESLint on the whole repository?

**Pull request merged**